web/site/stores/business.ts

```diff
@@ -85,6 +85,7 @@ export const useBusinessStore = defineStore('bar-sbc-business-store', () => {
 
     if (response) {
       payStatus.value = response.filing.header.status
+      arStore.arFiling = response
```

Comment:
need to update arFiling object in store so documents are updated to use inside the DocumentDownload component